composer.json

```diff
@@ -29,7 +29,10 @@
         "php-http/discovery": "^1.0",
         "php-http/client-common": "^1.4",
         "php-http/message-factory": "^1.0",
-        "php-http/client-implementation": "^1.0"
+        
+        "php-http/client-implementation": "^1.0",
+        "php-http/message-implementation": "^1.0",
+        "php-http/message-factory-implementation": "^1.0"
```

Comment:
Hum, looks like it is now ? https://packagist.org/providers/php-http/message-factory-implementation

But that's strange, as there is nothing on the `php-http/message` repo that says that it implements this virtual package, unless I read it wrong ?